includes/api/class-wc-rest-system-status-controller.php

```diff
@@ -722,6 +722,12 @@ public function get_database_info() {
 		);
 
 		foreach ( $database_table_sizes as $table ) {
+			// Only include tables matching the prefix of the current site, this is to prevent displaying all tables on a MS install not relating to the current.
+			if ( is_multisite() ) {
+				if ( 0 !== strpos( $table->name, $wpdb->prefix ) ) {
```

Comment:
I think this logic needs some tweaking. [Sites created after the first one will have `wp_2`, `wp_3`, etc. as their prefix.](https://premium.wpmudev.org/forums/topic/table-prefix-on-multisite#post-324657) 

This check will make the first site in the multisite (with prefix `wp_`) to show all tables from all sites, as `0 === strpos( 'wp_2_tablename', 'wp_' )`. I was able to reproduce this in my local environment.

Reply:
This is a tricky one as all prefixes start with the same chars, I was under the impression on a MS the first blog would prefix _1 as it uses the ID for all other sites in a network but guess this is not the case.

Any recommendation on what to do here?

Reply:
@claudiulodro thanks for pointing me in the right direction here, found there's a $wpdb->tables() function that simplifies this even more so used that instead.